machines/9x15_no_dram/Makefile.machine.include

```diff
@@ -1,8 +1,9 @@
-BSG_GLOBAL_X            = 9
-BSG_GLOBAL_Y            = 16
-BSG_VCACHE_SET          = 512
-BSG_VCACHE_WAY          = 2
-BSG_VCACHE_BLOCK_SIZE   = 8
-BSG_DRAM_SIZE           = 536870912  # 2^29 words (2GB)
-BSG_DRAM_BANK_SIZE      = 33554432
-BSG_DRAM_INCLUDED       = 0
+BSG_MACHINE_GLOBAL_X                  = 9
+BSG_MACHINE_GLOBAL_Y                  = 16
+BSG_MACHINE_VCACHE_SET                = 512
+BSG_MACHINE_VCACHE_WAY                = 2
+BSG_MACHINE_VCACHE_BLOCK_SIZE_WORDS   = 8
+BSG_MACHINE_DRAM_SIZE_WORDS           = 536870912  # 2^29 words (2GB)
+BSG_MACHINE_DRAM_BANK_SIZE_WORDS      = 33554432
+BSG_MACHINE_DRAM_INCLUDED             = 0
+BSG_MACHINE_MAX_EPA_WIDTH             = 28
```

Comment:
I realize this is the whole point of this pr, but why is it fixed though? :p I can chat with you tomorrow...